config/settings.yml

```diff
@@ -191,16 +191,17 @@ evss:
   pciu_address:
     timeout: 30
   mock_claims: false
-  mock_letters: false
+  mock_common_service: true
+  mock_disabilities_form: true
   mock_gi_bill_status: false
+  mock_itf: true
+  mock_letters: false
   mock_pciu: true
   mock_pciu_address: false
   mock_ppiu: true
-  mock_reference: false
-  mock_itf: true
-  mock_disabilities_form: true
-  mock_common_service: true
+  mock_reference: true
   mock_vso_search: false
+
```

Comment:
Maybe not a big deal but not sure if you meant to include this blank line since the aws block below is still a part of the evss settings.